logger.yaml

```diff
@@ -4,7 +4,7 @@ disable_existing_loggers: True
 formatters:
   default:
     format: '%(asctime)s %(message)s'
-    datefmt: '%X'
+    datefmt: '%Y-%m-%d %I:%M:%S %p'
```

Comment:
I think its better to leave %X and provide some way to override it, or just do assume unchanged in git. I think everyone will have different preference here, and it will end up being changed over and over. Or maybe we could make logger.yaml.example ?

Reply:
you are completely right, this change was a fault 😅